packages/dapp-svelte-wallet/react-ui/package.json

```diff
@@ -0,0 +1,135 @@
+{
+  "name": "react-ui",
```

Comment:
Please name this something like `@agoric/dapp-react-wallet-ui`.

Reply:
Done